src/de/deserializer.zig

```diff
@@ -270,21 +270,19 @@ pub fn Deserializer(comptime dbt: anytype, comptime Reader: type) type {
                             return error.Overflow;
                         }
 
-                        break :blk try parseInt(Value, slice);
+                        return visitor.visitInt(ally, De, try parseInt(Value, slice));
                     }
 
                     // If the visitor is not producing an integer, default to
                     // deserializing a 128-bit integer.
-                    break :blk try switch (slice[0]) {
-                        '0'...'9' => parseInt(u128, slice),
-                        else => parseInt(i128, slice),
-                    };
+                    switch (slice[0]) {
+                        '0'...'9' => return visitor.visitInt(ally, De, try parseInt(u128, slice)),
+                        else => return visitor.visitInt(ally, De, try parseInt(i128, slice)),
+                    }
```

Comment:
Missing some `try`s here.

```suggestion
                    return try switch (slice[0]) {
                        '0'...'9' => visitor.visitInt(ally, De, try parseInt(u128, slice)),
                        else => visitor.visitInt(ally, De, try parseInt(i128, slice)),
                    }
```